lib/session.js

```diff
@@ -0,0 +1,47 @@
+const db = require('./db');
+const { forEachSerialP } = require('omnibelt');
+
+const createSession = async () => {
+  const beforeCommits = [];
+  const afterCommits = [];
+  const onAborts = [];
+  if (!db._client) {
+    await db.init();
+  }
+  const session = await db._client.startSession();
+
+  const abortTransaction = async (error) => {
+    await forEachSerialP(async (afterAborted) => {
+      await afterAborted(error);
+    }, onAborts);
+    await session.abortTransaction();
+    await session.endSession();
+  };
+
+  return {
+    get: () => {
+      return session;
+    },
+    startTransaction: () => {
+      return session.startTransaction();
+    },
+    end: () => {
+      return session.end();
+    },
+    commitTransaction: async () => {
+      await session.commitTransaction();
+      await session.endSession();
```

Comment:
whats the difference between `session.endSession` and `session.end` ?

Reply:
I think I originally made end as a wrapper and then I switched it around and never actually ended up using it. so I'll just delete it.